packages/sd-jwt/src/action-handler.ts

```diff
@@ -186,8 +188,33 @@ export class SDJwtPlugin implements IAgentPlugin {
     const decodedVC = await sdjwt.decode(`${data}.${signature}`)
     const issuer: string = ((decodedVC.jwt as Jwt).payload as Record<string, unknown>).iss as string
     const verifierKey: SignKeyResult = await this.getSignKey({ identifier: issuer, vmRelationship: 'verificationMethod' }, context)
-    const key = await context.agent.keyManagerGet({ kid: verifierKey.key.jwkThumbprint ?? verifierKey.key.kid })
-    return this.algorithms.verifySignature(data, signature, toJwk(key.publicKeyHex, key.type))
+    const x5c: string[] | undefined = (decodedVC.jwt as Jwt).header?.x5c as string[]
+    const x5t: string | undefined = (decodedVC.jwt as Jwt).header?.x5t as string
+    let publicKey: string | undefined
+
+    if (x5c) {
+      const certChain = x5c.map((cert) => `-----BEGIN CERTIFICATE-----\n${cert}\n-----END CERTIFICATE-----`)
```

Comment:
This method can handle ders and pems. No need to do this manual conversion. Besides we have more robust methods to do that.

Reply:
changed this to just pass in the first one of the chain